lib/src/main/java/co/nimblehq/alpine/lib/nfc/ImageUtil.kt

```diff
@@ -0,0 +1,119 @@
+package co.nimblehq.alpine.lib.nfc
+
+import android.content.Context
+import android.graphics.*
+import android.util.Base64
+import co.nimblehq.alpine.lib.model.Image
+import jj2000.j2k.decoder.Decoder
+import jj2000.j2k.util.ParameterList
+import org.jmrtd.lds.AbstractImageInfo
+import org.jnbis.WsqDecoder
+import java.io.*
+import kotlin.experimental.and
+
+object ImageUtil {
```

Comment:
Shall we make this `internal`? 😄 
```suggestion
internal object ImageUtil {
```